### ISSUE TITLE
no ./configure in master?

### ISSUE THREAD

**Issue by entaylor** [astromatic/swarp#10]
I find myself unable to install the swarp master branch, having downloaded a .zip archive.

I see that there is the configure.ac, but no ./configure .  I also see that there is a ./configure in, e.g., the RoyBranch.

I think either the ./configure needs to be added to the branch, or the INSTALL needs to be edited to explain how to obtain and use autoconf.

This is a minor issue, but it has taken me maybe 30 min to discover autoconf.  Which is maybe good for me, i concede, but ... well, not strictly necessary.

**Comment by kirxkirx** [comment]
Similar story here: I want to compile swarp on a machine with no autoconf (but with all the other usual developer tools). Having an actual configure script at least in the release archive would be helpful.

**Comment by marie-lougm** [comment]
Same issue here. I tried autoreconf -vif  and then ./configure and now I have an error about cfitsio:
`checking for cc_r... gcc
checking for fitsio.h... no
checking for cfitsio/fitsio.h... no
configure: error: CFITSIO include files not found at default location! Exiting.`

Don't know how to proceed now...

**Comment by ebertin** [comment]
You're right, this is an oversight and it needs to be properly documented. You have to run the `./autogen.sh` script that comes with the package, that will regenerate a proper `configure` script for you. The reason for this is that the autotools evolve over time (or at least they did), and in many situations  the original `configure` would fail. Never had any issue with `autogen.sh` so far., at least on Linux.
